cmd/context/create.go

```diff
@@ -172,8 +172,19 @@ func (c *ContextCommand) UseContext(ctx context.Context, ctxOptions *ContextOpti
 			return err
 		}
 
-		if !hasAccess {
-			return oktetoErrors.UserError{E: fmt.Errorf("namespace '%s' not found on context '%s'", ctxOptions.Namespace, ctxOptions.Context),
+		// if using a new context, our cached namespace may have been removed
+		// so swap over to the personal namespace instead of erroring
+		if !hasAccess && !ctxOptions.FromNamespace {
+			oktetoLog.Success(
```

Comment:
Could you provide a screenshoot of how this looks like?

I guess we should be using `oktetoLog.Warning` here, but I want to see how it looks like

Reply:
Sure, here is the before and after.
![image](https://user-images.githubusercontent.com/8585244/215582119-ef59d88b-bca9-4d01-b614-a523a0ee83c2.png)

Reply:
I can swap over logging levels if you want

Reply:
Yeah I agree with @jLopezbarb , I think we should show a `Warning` or `Info` here

Reply:
I think we should replace it with a warning message. I think that using success hides the meaning of the message, which is that the user should know why we are redirecting him to his personal namespace. WDYT?

Reply:
Done